bentoml/_internal/utils/tensorflow.py

```diff
@@ -163,9 +165,66 @@ def normalize_spec(value: t.Any) -> "tf_ext.TypeSpec":
     raise BentoMLException(f"Unknown type for tensor spec, got{type(value)}.")
 
 
+def cast_py_args_to_tf_function_args(
+    signature: list[tf_ext.TensorSpec],
+    *args: t.Any,
+    **kwargs: t.Any,
+) -> tuple[t.Any, ...]:
+    """
+    Cast python arguments (args, kwargs) to tensorflow function arguments.
+
+    Args:
+        signature (:code:`list[tf.TensorSpec]`):
+            signature of the tensorflow function.
+        *args (:code:`t.Any`):
+            positional arguments of the Python function.
+        **kwargs (:code:`t.Any`):
+            keyword arguments of the Python function.
+    """
+    import inspect
```

Comment:
do we need to put inspect import here? It's a builtin module so we don't need to lazy import it?

Reply:
I think lazy importing works too because it's builtin

Reply:
inspect is fast to import and no need to lazily import it.

Reply:
@aarnphm 
Then are there any cons of lazily importing?

Reply:
utils functions rarely have dependencies overlap by might be imported everywhere. Thus lazy import by default should be good for them.

Reply:
just for readability. Usually it is not as needed to do unless the library is very heavy to import.

Reply:
What readability?

Reply:
don't worry. It is preference only.

Reply:
We should probably be more structured about lazy importing for sure.